x-pack/packages/kbn-cloud-security-posture/src/hooks/use_navigate_findings.test.ts

```diff
@@ -6,14 +6,15 @@
  */
 
 import { renderHook, act } from '@testing-library/react-hooks/dom';
-import { useNavigateFindings, useNavigateVulnerabilities } from './use_navigate_findings';
+import { useNavigateVulnerabilities } from './use_navigate_findings';
+import { useNavigateFindings } from './use_navigate_findings';
```

Comment:
seems like the change is done by mistake, two imports from the same file

Reply:
Woops, you are right